src/components/link/link.style.js

```diff
@@ -78,15 +78,21 @@ const LinkStyle = styled.span`
   `}
 `;
 
-LinkStyle.defaultProps = {
+StyledLink.defaultProps = {
   theme: baseTheme,
   disabled: false,
   hasContent: true,
 };
 
-LinkStyle.propTypes = {
+StyledLink.propTypes = {
   disabled: PropTypes.bool,
   hasContent: PropTypes.bool,
 };
 
-export default LinkStyle;
+const StyledContent = styled.span``;
+
+StyledContent.defaultProps = {
```

Comment:
This doesn't need the theme prop as it isn't used anywhere at the moment, you can just remove this `defaultProps` altogether

Reply:
good point, changed, thanks